doc/md/monitoring/05-helm.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Deploying with Helm
+---
+Prerequisites:
+- An Atlas Cloud account
+- A Kubernetes cluster
+- An API key for your agent
+- Optional: Password to the database you want to connect to
+
+### Create a namespace for the agent
+
+```shell
+kubectl create namespace atlas-agent
+```
+
+### Create a secret containing sensitive information
+
+Create a `Secret` named `atlas-agent` with your API key,
+
+```shell
+kubectl create secret generic atlas-agent --from-literal api-key=YOUR_API_KEY
+```
+
+If you wish the Agent to have access to your database using a password, create a second secret named
+`atlas-agent-db` with your database password:
+
+```shell
+kubectl create secret generic atlas-agent-db --from-literal password=YOUR_DB_PASSWORD
+```
+
+### Install the agent
+
+Create a file named `values.yaml` with the following content:
+
+```yaml
```

Comment:
pls add `yaml` here, because it doesn't have the temere by default.

https://github.com/ariga/atlas/blob/808503439899ba286833472456d29cd9fe6226ec/doc/website/docusaurus.config.js#L16

Reply:
Are you sure? It looks alright when used.

Reply:
https://github.com/FormidableLabs/prism-react-renderer/blob/f87355c252595b0300a822d5767fb6d1fab3c300/packages/generate-prism-languages/index.ts#L20